CHANGELOG.md

```
@@ -0,0 +1,25 @@
+- 2025-09-19 — **Evaluate markdown component library** (hubless/m0/task/0005) — Tested, Documented, Shipped
+
+## Completed Stories (For Reference)
+
+<!-- Generated by docs-components; do not edit manually. -->
+| ID | Title | Completed Status |
+| --- | --- | --- |
+| — | — | — |
+
+## Historical Archives
+
+See `@hubless/issues/generated/archive.md` for the full backlog of completed stories and tasks.
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)